### ISSUE TITLE
[IconButton] Accessibility Guidance

### ISSUE THREAD

**Issue by cgatian** [radix-ui/themes#643]
I noticed the docs for `IconButton` do not supply guidance on adding an `aria-label` to the `IconButton` component.

I think there should be a callout that the provided examples are not accessible to screen readers, and when using `IconButton` the appropriate `aria-label` attribute should be supplied.

**Comment by balazsotakomaiya** [comment]
Opened a PR for this here :)

https://github.com/radix-ui/website/pull/844

**Comment by cgatian** [comment]
Nice. Thank you!
